### ISSUE TITLE
[Bug]: www.photopea.com

### ISSUE THREAD

**Issue by ema7569** [Mikubill/sd-webui-controlnet#2422]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

It is possible to remove this thing from controlnet? It show a button and open a frame to the site www.photopea.com to edit image in a private web site. This is not open source and must be a separated solution like any external system or extension of this type. I not understand if the way for controlnet is to start to be a private/paid model or if it's a hack from a community dev. 

### Steps to reproduce the problem

...

### What should have happened?

...

### Commit where the problem happens

...

### Command Line Arguments

```Shell
...
```

### List of enabled extensions

...

### Console logs

```Shell
...
```

**Comment by huchenlei** [comment]
See discussion here: https://github.com/AUTOMATIC1111/stable-diffusion-webui-extensions/issues/255

**Comment by huchenlei** [comment]
TLDR is yes. You disable in settings > controlnet > disable photopea edit.
